scripts/clang-tidy-only-diff.sh

```diff
@@ -41,20 +41,25 @@ if [ ${#modified_filepaths[@]} = 0 ]; then
   exit 0
 fi
 
+readonly num_jobs="${1:-$(nproc)}"
```

Comment:
nproc won't work on mac. Also should we just use 4 by default?

Reply:
the bash script that this sources sets an NPROC env var so that that shoudl work on mac. seems we just forgot to use it here

Reply:
Oh nice, amending.